docs/rules/index.md

```diff
@@ -5,42 +5,45 @@ layout: doc
 <!-- Note: No pull requests accepted for this file. See README.md in the root directory for details. -->
```

Comment:
Ah. Sorry, this file will have to be changed in ESLint main repository. It gets copied here automatically. I'm also not 100% sure that jekyll supports adding `span` into markdown.